index.html

```diff
@@ -0,0 +1,397 @@
+                  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 132 97" height="32" width="36">
+                  <path fill="currentColor" fill-rule="evenodd"
+                    d="M25 30.2c3.5 1.5 7.7-.2 9.1-3.7 1.5-3.6-.2-7.8-3.9-9.2-3.6-1.4-7.6.3-9.1 3.9-1.4 3.5.3 7.5 3.9 9zM9 39.5c3.6 1.5 7.8-.2 9.2-3.7 1.5-3.6-.2-7.8-3.9-9.1-3.6-1.5-7.6.2-9.1 3.8-1.4 3.5.3 7.5 3.8 9zM4.4 57.2c3.5 1.5 7.7-.2 9.1-3.8 1.5-3.6-.2-7.7-3.9-9.1-3.5-1.5-7.6.3-9.1 3.8-1.4 3.5.3 7.6 3.9 9.1zm38.3-21.4c3.5 1.5 7.7-.2 9.1-3.8 1.5-3.6-.2-7.7-3.9-9.1-3.6-1.5-7.6.3-9.1 3.8-1.3 3.6.4 7.7 3.9 9.1zm64.4-5.6c-3.6 1.5-7.8-.2-9.1-3.7-1.5-3.6.2-7.8 3.8-9.2 3.6-1.4 7.7.3 9.2 3.9 1.3 3.5-.4 7.5-3.9 9zm15.9 9.3c-3.6 1.5-7.7-.2-9.1-3.7-1.5-3.6.2-7.8 3.7-9.1 3.6-1.5 7.7.2 9.2 3.8 1.5 3.5-.3 7.5-3.8 9zm4.7 17.7c-3.6 1.5-7.8-.2-9.2-3.8-1.5-3.6.2-7.7 3.9-9.1 3.6-1.5 7.7.3 9.2 3.8 1.3 3.5-.4 7.6-3.9 9.1zM89.3 35.8c-3.6 1.5-7.8-.2-9.2-3.8-1.4-3.6.2-7.7 3.9-9.1 3.6-1.5 7.7.3 9.2 3.8 1.4 3.6-.3 7.7-3.9 9.1zM69.7 17.7l8.9 4.7V9.3l-8.9 2.8c-.2-.3-.5-.6-.9-.9L72.4 0H59.6l3.5 11.2c-.3.3-.6.5-.9.9l-8.8-2.8v13.1l8.8-4.7c.3.3.6.7.9.9l-5 15.4v.1c-.2.8-.4 1.6-.4 2.4 0 4.1 3.1 7.5 7 8.1h.2c.3 0 .7.1 1 .1.4 0 .7 0 1-.1h.2c4-.6 7.1-4.1 7.1-8.1 0-.8-.1-1.7-.4-2.4V34l-5.1-15.4c.4-.2.7-.6 1-.9zM66 92.8c16.9 0 32.8 1.1 47.1 3.2 4-16.9 8.9-26.7 14-33.5l-9.6-3.4c1 4.9 1.1 7.2 0 10.2-1.5-1.4-3-4.3-4.2-8.7L108.6 76c2.8-2 5-3.2 7.5-3.3-4.4 9.4-10 11.9-13.6 11.2-4.3-.8-6.3-4.6-5.6-7.9 1-4.7 5.7-5.9 8-.5 4.3-8.7-3-11.4-7.6-8.8 7.1-7.2 7.9-13.5 2.1-21.1-8 6.1-8.1 12.3-4.5 20.8-4.7-5.4-12.1-2.5-9.5 6.2 3.4-5.2 7.9-2 7.2 3.1-.6 4.3-6.4 7.8-13.5 7.2-10.3-.9-10.9-8-11.2-13.8 2.5-.5 7.1 1.8 11 7.3L80.2 60c-4.1 4.4-8 5.3-12.3 5.4 1.4-4.4 8-11.6 8-11.6H55.5s6.4 7.2 7.9 11.6c-4.2-.1-8-1-12.3-5.4l1.4 16.4c3.9-5.5 8.5-7.7 10.9-7.3-.3 5.8-.9 12.8-11.1 13.8-7.2.6-12.9-2.9-13.5-7.2-.7-5 3.8-8.3 7.1-3.1 2.7-8.7-4.6-11.6-9.4-6.2 3.7-8.5 3.6-14.7-4.6-20.8-5.8 7.6-5 13.9 2.2 21.1-4.7-2.6-11.9.1-7.7 8.8 2.3-5.5 7.1-4.2 8.1.5.7 3.3-1.3 7.1-5.7 7.9-3.5.7-9-1.8-13.5-11.2 2.5.1 4.7 1.3 7.5 3.3l-4.7-15.4c-1.2 4.4-2.7 7.2-4.3 8.7-1.1-3-.9-5.3 0-10.2l-9.5 3.4c5 6.9 9.9 16.7 14 33.5 14.8-2.1 30.8-3.2 47.7-3.2z">
+                  </path>
+                  <image
+                    src="/assets/govuk-logotype-crown-0cdbde943be3a6518a5d7c00b607d134f4d5e3d0e8c68a1bfa2dc81f1f3edc4e.png"
+                    class="govuk-header__logotype-crown-fallback-image"></image>
+                </svg>
+                <span class="govuk-header__logotype-text">
+                  GOV.UK
+                </span>
+              </span>
+              <span class="govuk-header__product-name gem-c-header__product-name">
+                Account
+              </span>
+            </a>
+          </div>
+
+        </div>
+        <div class="govuk-header__content gem-c-header__content">
+
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <div id="wrapper" class="govuk-width-container group">
```

Comment:
Just being pedantic really, I would probably restructure this a little bit to match the styling on the errors we have in static.
So I would remove the `<h1>` from the  `<div class="inner">`. Instead I would do something like 
```
<main id="content" role="main" class="group">
  <header class="page-header group">
    <div>
      <h1> There is a problem with GOV.UK accounts </h1>
    </div>
  </header>
  <div class="article-container">
    ...all that other stuff
  </div>
</main>
```

Just for consistency. Probably 🤷🏻‍♀️  
Hopefully people wouldn't be encountering all sorts of errors all across the site enough to notice discrepancies in the styling though.

Reply:
:D that's great and please pedant away! I'll get that fixed now

Reply:
:thinking: humm I've given this a go (will post a WIP commit here shortly). But the styling has gone a bit funky. I think this is why I reshuffled things int he first place.

I suspect I'm missing some gov.uk classes to make this all tidier?

my changes end up looking like this:
![image](https://user-images.githubusercontent.com/3694062/102766484-1b68ff80-4376-11eb-9c84-7da0ee768d03.png)

Reply:
https://github.com/alphagov/govuk-account-static-errors/pull/2/commits/1d4d17c95c85d7bf284d09949ad4736e2815653f

Reply:
Hmmm I am not sure what you mean by funky styling! to me it looks more similar to the static errors now. At least to the [404 one](https://www.gov.uk/nfirj4398r8938948) as that's the main one I can easily compare it to.

One thing you can do to add more spacing below the  `Status code: 503` is add a govuk spacing class to `article-container`. 
So `<div class="article-container">` becomes `<div class="article-container govuk-!-margin-bottom-9">`.
That should help balance it out visually.

Reply:
You are right!
I think i mean there's a lot of padding around that heading.

But I think i am finding it jarring because on the other pages the first paragraph is a bit more substantial than "try again later"

So the page looks a bit top heavy. But yeah it is consistent.

I'll give that margin tag a go and push again.